pkg/arvo/lib/group.hoon

```diff
@@ -0,0 +1,98 @@
+/-  *group, *metadata-store
```

Comment:
+1 for most of the functions here, these are good and useful! Looks like they all pull data from the store though, which makes me wonder why they're not in `/lib/group-store`. You might still leave the call-with-data equivalent in here, if doing those operations on group nouns themselves turns out to be common.

Also could consider not exposing the `-path` functions at all, if you want the dev/user to work with `group-id`s as much as possible.